### DISCUSSION TITLE
v2.0.0-alpha-2

### DISCUSSION THREAD

**Discussion by qier222** [qier222/YesPlayMusic#2014] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/qier222/YesPlayMusic/releases/tag/v2.0.0-alpha-2'>v2.0.0-alpha-2</a>.</em>

**Comment by serlar** [comment]
可以有心动播放模式吗

**Reply by HQYSteven** [reply]
deb rpm snap faltpak

**Comment by MadamaDadale** [comment]
这一版音质好爽，加油作者大大!

**Comment by georgemarx1024** [comment]
1. 希望显示模式可以提供多种并提供选择功能
2. 主界面希望把左下角的英文改成中文
3. 希望可以提供传统的网易云音乐显示模式
4. 希望可以提供多种主题配色

**Comment by alent17** [comment]
太漂亮了(๑•̀ㅂ•́)و✧

**Comment by Doraemon148** [comment]
为什么没有歌词呢，什么时候支持

**Comment by buchylx** [comment]
大佬加油，用了好久yesplaymusic了，非常赞，希望mac能加入桌面歌词显示，或者显示在顶部状态栏中

**Comment by zxmwd777** [comment]
找不到私人FM了啊。。。

**Comment by Heart-Beats** [comment]
希望可以加入后台播放功能，现在只能最小化播放

**Comment by Acorea3372** [comment]
找不到调节音量的地方了QAQ

**Reply by Rushman611** [reply]
可以尝试WIN+G打开Xbox Game Bar快捷调一下音量，就是比较麻烦罢了   还是建议用回YESPLAYMUSIC等完善吧

**Comment by Liume3653** [comment]
加油！

**Comment by why-qw1ko** [comment]
个人感觉界面没有原来的好看了

**Comment by co10rs** [comment]
这软件的音量怎么调整啊

**Comment by AppleYang530** [comment]
希望Browse界面的歌单也可以有个人歌单那种键鼠放上去显示歌单名的效果(>_<)

**Comment by JevonGit** [comment]
可以加个状态栏歌词吗？

**Comment by Ender-Wiggin2019** [comment]
这版本UI很漂亮呀，支持

**Reply by kafeibujiatan** [reply]
这个版本的很时尚

**Comment by YJL12345** [comment]
最大化圆角还在啊？win10，客户端里面没有页面看版本号，也没有可以点链接跳到github，不方便更新

**Comment by yuanjie408** [comment]
这版也能增加ARM64   linux版本么

**Comment by JaylinChai** [comment]
有个恶性bug 接多屏后，软件还是在原始的位置，没法回到主屏

**Comment by ts2077** [comment]
开启了TUN MODE 很多版权音乐还是无法听，开了全局也不行，我的节点是正常使用的

**Comment by YiFan04** [comment]
加油  希望能把vip歌曲拿下

**Comment by uigonezjjj** [comment]
网页二维码扫不了

**Comment by lupoCW-71** [comment]
一个小建议：可以加入白色界面的UI吗，纯黑的界面对于散光人士有点难以看清字

**Comment by ZAC3K** [comment]
很喜欢新的软件的界面设计，简洁清爽
问题1. 网易云日推歌单不见了，只剩私人FM，个人认为作为官方算法做出来的歌单可否和用户自己创建的热门歌单区分一下
问题2.在播放小窗那边播放器把专辑封面挡住了
<img width="349" alt="image" src="https://user-images.githubusercontent.com/132423630/235831376-87bc2e0e-99e9-45e9-8fee-7a87216ad65a.png">
问题3.设置单独放了一个小齿轮按钮在右上角，旁边头像点开的次栏中又有一个设置按钮，感觉有些重复
问题4.在macos程序坞中大小不会缩放，比周围图标显得大一圈
最后还是对大佬表示支持，很喜欢一些小但是不花里胡哨的动画，很丝滑！

**Comment by wudangshanus** [comment]
歌曲喜欢的那个爱心 可以点亮变成红色的吗

**Comment by xxxx1024** [comment]
Mac 版，希望能够后台播放，增加歌曲，现在很多歌曲都不能放。

**Comment by zhangdzhang** [comment]
在低分屏下打开界面太大超出了...OS: linux mint 21

**Comment by Mr-ZMH** [comment]
窗口显示界面的圆角半径太大了，跟其他窗口一起好违和

**Reply by Mr-ZMH** [reply]
还有最大化窗口会盖住一丝丝任务栏

**Comment by Faxmell** [comment]
扫码登陆不能用，没反应

**Comment by sien75** [comment]
真舒服,比第一版舒服! 歌词模块已经等不及啦

**Comment by xxxx1024** [comment]
每日推荐歌单在哪里？没有找到
如果没有，希望后面能够加上这个功能。

**Comment by jiangbai2022** [comment]
希望把随机播放和循环播放放在一起轮播，UnblockNeteaseMusic的qq cookie填上并不可用

**Comment by xxxx1024** [comment]
新老版本都可以播放大部分网易云vip歌曲的替代版本，同一首vip歌曲2个软件播放的版本不一样，经常播放的版本效果都不太好，希望可以解决。

**Comment by Oswald314** [comment]
这个有快捷方式吗？为啥是网页版的啊

**Reply by jiangbai2022** [reply]
有啊，在仓库里下载对应文件，https://github.com/qier222/YesPlayMusic/releases

**Reply by cetsupport** [reply]
现在不提供deb格式下载了吗？

**Comment by DyingSecret** [comment]
不能复制歌曲链接，分享后经常复制到的是专辑或者播放列表的链接

**Comment by lovelylion-jojo** [comment]
新版的界面很喜欢😍，期待后续完整版的样子

**Comment by ldpc520** [comment]
并不能扫码或帐号密码登陆了

**Comment by XD621** [comment]
安装后无法登录，界面没有任何歌曲信息，偶尔还会闪退

**Comment by SHMG8816** [comment]
1.播放MV的时候按进度条会直接跳到最后一秒结束播放。
2.MY Music-歌单 鼠标指到哪个歌单封面就会dim（暗黑）并显示歌单名字，希望能应用到Discover页面和Browse页面上。（指到专辑就dim并显示专辑名称与歌手）

**Comment by 24karas** [comment]
> _此讨论是从版本 [v2.0.0-alpha-2](https://github.com/qier222/YesPlayMusic/releases/tag/v2.0.0-alpha-2) 创建的。_

1.不支持软件内调整音量；
2.不支持歌单名称显示；
3.不支持歌词显示；

**Comment by cxy1658293650** [comment]
不能播VIP歌曲我要这干啥，还不如用原版。。。

**Comment by YunX-a** [comment]
希望增加其他界面ui以及后台播放

**Comment by YangHaiming** [comment]
R3PLAY 
无法定位程序输入点getpackagefamilyname于动态链接库KERNEL32.DLL上
windows 7旗舰版

**Reply by ivanivanivanivanivanivan** [reply]
R3PLAY-2.0.0-linux.AppImage 在Ubuntu上如何安装呢？

**Reply by ogios** [reply]
@ivanivanivanivanivanivan 直接当作一个可执行程序运行就可以，我这边是这样，不过还要多下一个fuse2的包

**Comment by ogios** [comment]
很喜欢，只不过随机播放还是开发中，这是我觉得非常重要的一个功能，还有音量调整没找到(?)
流畅程度感觉也比0.4.7好很多

**Comment by pang-xf** [comment]
好久没更新了 催更

**Comment by Cherub1Zhuang** [comment]
新版本设想肯定比原来的好，功能肯定一更多，但是目前新版本开发的功能太少，bug太多，很多还没有开发完，还不如原来的，而且我个人而言，还是更喜欢原来浅色调的风格，希望可以加入浅色和深色主题切换功能，感谢你们的奉献

**Comment by End782** [comment]
喜欢新版的界面:heart_eyes:, 期待正式版

**Comment by CaoBo1024** [comment]
希望增加本地音乐播放， 并添加显示本地歌词的功能。

**Comment by yukunjie** [comment]
windows希望加一下关闭窗口时最小化到系统托盘的功能。

**Comment by rqgbqd** [comment]
TypeError: Cannot read properties of undefined (reading 'artists')，比如搜索“上师“。

**Comment by nightknight3487** [comment]
希望可以显示苹果音乐的专辑封面，还有，为什么我在歌手页面无法查看所有专辑

**Comment by youlan2333** [comment]
圆角看起来有些奇怪，可否加一个开关来切换窗口的矩形或圆角外观呢？

**Comment by Shaddoll126** [comment]
windows希望加一下关闭窗口时最小化到系统托盘的功能。

**Comment by bantanglady** [comment]
希望能支持播放本地的音乐文件以及关联到云端的歌词，谢谢大佬！

**Comment by gaivrt** [comment]
这个版本UI真的好好好好看，大佬加油加油。

**Comment by Theoshen** [comment]
未来会有 docker 版本吗

**Comment by EmeraldDryad** [comment]
请问有手机端的吗

**Reply by RikkAsh-k** [reply]
应该是没有的

**Comment by RikkAsh-k** [comment]
歌词显示什么时候能做出来啊😭

**Comment by pppeace** [comment]
这个 2.0 版本还更新吗？

**Reply by pppeace** [reply]
🙃

**Comment by crisp2466** [comment]
![image](https://github.com/qier222/YesPlayMusic/assets/59132817/86ceadb1-026d-40b4-98b7-7c48ace59ba1)
幻13用户很痛苦... 1920*1200 150%
